core/src/main/java/com/linecorp/armeria/internal/common/ArmeriaHttpUtil.java

```diff
@@ -696,6 +693,8 @@ public static HttpHeaders toArmeria(io.netty.handler.codec.http.HttpHeaders inHe
 
     /**
      * Converts the specified Netty HTTP/1 headers into Armeria HTTP/2 headers.
+     * Functionally, this method is expected to behavior in the same way as
+     * {@link #purgeHttp1OnlyHeaders(io.netty.handler.codec.http.HttpHeaders, HttpHeadersBuilder)}.
      */
```

Comment:
Note that I haven't migrated this method to use `purgeHttp1OnlyHeaders` since this is probably slower for other code paths using this method without zero-copy